pages/_app.tsx

```diff
@@ -28,6 +28,7 @@ import Layout from 'ui/shared/layout/Layout';
 import Web3ModalProvider from 'ui/shared/Web3ModalProvider';
 
 import 'lib/setLocale';
+import 'ui/games/Capybara.css';
```

Comment:
Can these styles be loaded only when a specific screen is displayed? I don't think they should load on every page.